src/api/error.jl

```diff
@@ -35,9 +53,9 @@ const SHORT_ERROR = Ref(true)
 
 function Base.showerror(io::IO, err::H5Error)
     n_total = length(err)
-    print(io, "H5Error: ", err.msg)
+    print(io, "$(typeof(err)): ", err.msg)
```

Comment:
I used `typeof` here so that the full module path is printed:
![image](https://user-images.githubusercontent.com/4319522/123172841-b472f300-d44b-11eb-8907-4e4217784773.png)


I think this is a good idea in acse we decide to use the same exception error name in the main module.